src/rpc/net.cpp

```diff
@@ -601,8 +601,8 @@ static RPCHelpMan getnetworkinfo()
                                 {RPCResult::Type::BOOL, "proxy_randomize_credentials", "Whether randomized credentials are used"},
                             }},
                         }},
-                        {RPCResult::Type::NUM, "relayfee", "minimum relay fee for transactions in " + CURRENCY_UNIT + "/kB"},
-                        {RPCResult::Type::NUM, "incrementalfee", "minimum fee increment for mempool limiting or BIP 125 replacement in " + CURRENCY_UNIT + "/kB"},
+                        {RPCResult::Type::NUM, "relayfee", "minimum relay feerate for transactions in " + CURRENCY_UNIT + "/kvB"},
+                        {RPCResult::Type::NUM, "incrementalfee", "minimum feerate increment for mempool limiting or BIP 125 replacement in " + CURRENCY_UNIT + "/kvB"},
```

Comment:
ISTM the user-facing rpc and qt mostly use "fee rate"

Reply:
Thanks, fixed